src/safeposix/syscalls/net_calls.rs

```diff
@@ -2834,6 +2834,40 @@ impl Cage {
         }
     }
 
+    /// ## ------------------SOCKETPAIR SYSCALL------------------
+    /// ### Description
+    /// The `socketpair_syscall()` call creates an unnamed pair of connected sockets in the specified domain, of the specified type, and using
+    /// the optionally specified protocol.
+    /// The file descriptors used in referencing the new sockets are returned in sv[0] and sv[1]. The two sockets are
```

Comment:
while the man page does say the new sockets are in sv[0] and sv[1], it seems to be worth changing that to sv.sock1 and sv.sock2 as thats our implementation